lib/puppet/provider/package/apk.rb

```diff
@@ -38,6 +38,7 @@ def query
     self.class.instances.each do |provider|
       return provider.properties if name.downcase == provider.name.downcase
     end
+    return nil
```

Comment:
Errr. I'm not a ruby expert, but if that return isn't there I'm getting errors later down the line (because it isn't returning 'nil' by default).

>f2b7ac4aa604:/# ruby --version
ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-linux-musl]